### PR TITLE
Make legend entries into links. See #308

### PR DESCRIPTION
**Pull request by peterblazejewicz**
- links into category if available
- if there is no directory, links to the first relevant entry within
  page (e.g. 5V category)

Things to rework in future:
- category for GPIO, 5V, 3.3V

Thanks!

image

### CODE REVIEW

File: common/page.html [hunk 1]
```diff
@@ -6,13 +6,48 @@
...
+							</a>
+						</li>
+						<li class="pow5v">
+							<a href="/pinout/pin2_5v_power" title="5v (Power)">
+								<span class="default"></span>
+								<span class="pin"></span> 5v <small>(Power)</small>
+							</a>
+						</li>
+						<li class="pow3v3">
+							<a href="/pinout/pin1_3v3_power" title="5v (Power)">
```
Comment (lurch) [inline common/page.html]: I think this should be "3.3v (Power)" ?

File: common/page.html [hunk 2]
```diff
@@ -6,13 +6,48 @@
...
 					<ul>
-						<li class="gpio"><span class="pin"></span> GPIO <small>(General Purpose IO)</small></li>
-						<li class="spi"><span class="pin"></span> SPI <small>(Serial Peripheral Interface)</small></li>
-						<li class="i2c"><span class="pin"></span> I<sup>2</sup>C <small>(Inter-integrated Circuit)</small></li>
-						<li class="uart"><span class="pin"></span> UART <small>(Universal Asyncronous Receiver/Transmitter)</small></li>
-						<li class="gnd"><span class="pin"></span> Ground</li>
-						<li class="pow5v"><span class="pin"></span> 5v <small>(Power)</small></li>
-						<li class="pow3v3"><span class="pin"></span> 3v <small>(Power)</small></li>
+						<li class="gpio">
+							<a href="/pinout/wiringpi" title="GPIO (General Purpose IO)">
```
Comment (lurch) [inline common/page.html]: Why link to the WiringPi page?

File: common/page.html (hunk 1 above)
Comment (peterblazejewicz) [reply common/page.html]: yes, thanks for spotting this! updated

File: common/page.html (hunk 2 above)
Comment (peterblazejewicz) [reply common/page.html]: yeah, that's looks like arbitrary choice. I've picked this up not being able to decide where to point this link. There is no `gpio` topic page atm.

**Comment by Gadgetoid** [comment]
Thank you- great idea! Merged.

**Pull request merged**